app/helpers/column_sorting_helper.rb

```diff
@@ -30,6 +31,43 @@ def sortable(column, title = nil, path = :url_for)
             { :class => css_class }
   end
 
+  def th_sortable(column, title = nil, path = :url_for)
```

Comment:
This will eventually replace the old method `sortable` once all tables have been implemented using PF4.

Reply:
I think this might need a unit test?

Reply:
![](https://media.giphy.com/media/JWCahAf61Z43e/giphy.gif)

In all seriousness, I don't think it's worth it. The method (and the original) are to fiddly and IMHO it should be enough to test it indirectly (via 🥒)